oem/ibm/configurations/pdr/ibm,bonnell/4.json

```diff
@@ -0,0 +1,420 @@
+{
+ "sensorPDRs": [
+   {
+    "pdrType" : 4,
+    "entries" : [{
+        "type" : 45,
+        "instance" : 1,
+        "container" : 1,
+        "sensors" : [{
+            "set" : {
+                "id" : 17,
+                "size" : 1,
+                "states" : [1,2]
+            },
+            "dbus" : {
+                "path": "/xyz/openbmc_project/led/groups/enclosure_identify",
```

Comment:
All the dbus details are wrong. This should match with "/xyz/openbmc_project/led/physical/virtual_enc_id". Please check the 4.json file at "pldm/oem/ibm/configurations/pdr/ibm,rainier-1s4u" as an example.